src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -134,6 +149,7 @@ export const DropdownMenu: React.FC<Props> = ({
       <div className={componentClassName} {...other}>
         <ul
           className={styles['dropdown-menu__list']}
+          onClick={handleOnClick}
```

Comment:
Can you explain this part to me? I'm curious if it could possibly be overriding the click behavior on the items. I went to the course planner in storybook in the chromatic build for this PR, and the items in the dropdown list for moving the cards around aren't doing anything for me.

Reply:
Since `buttonDropdown` controls the active/open state for `DropdownMenu` this calls the `handleOnClick` callback which closes sets it closed in `buttonDropdown`.

I don't think any of the buttons in the dropdown in the course planner edit page in EDS had attached functionality to begin with as they don't work in https://chanzuckerberg.github.io/edu-design-system/?path=/story/pages-course-planner-edit--default and lack of `onClick` being passed in https://github.com/chanzuckerberg/edu-design-system/blob/main/.storybook/pages/CoursePlannerEdit/CoursePlannerEdit.tsx#L288-L332

Reply:
Awesome, thanks!